### DISCUSSION TITLE
Intercom Bitron AV1407/010 stuck on open

### DISCUSSION THREAD

**Discussion by ghost** [PricelessToolkit/ESPBell-LITE#13] [Q&A] [answered]
Hello,
Thanks for this project!

I could use some help: I have a [Bitron AV1407-010 Universal T-Line Intercom Handset Data Sheet.pdf](https://github.com/PricelessToolkit/ESPBell-LITE/files/15312053/Bitron.AV1407-010.Universal.T-Line.Intercom.Handset.Data.Sheet.pdf).

I wired it like this:
![imgonline-com-ua-dexifTK1QncH7f8cm](https://github.com/PricelessToolkit/ESPBell-LITE/assets/7684370/846c440b-c07b-4fef-822b-5b6cdaa5a14c)

from the manual:
![image](https://github.com/PricelessToolkit/ESPBell-LITE/assets/7684370/e5ee2df9-29dc-435b-8678-a3aa8ac4cd7d)

From left to right of ESPBell-LITE
|ESPBell-LITE | intercom|
|--|--|
|LOCK (black) | 6 (system ground)|
|LOCK (red) | OP (To be connected to the door phone terminal pin 6) **disconnected in picture** (see below)|
|... | not connected|
|BELL (green) | 6 (system ground)|
|BELL (yellow) | APP (call tone by buzzer or electronic)|
|VCC (black) | V (of a usb cable)|
|GND (red) | G (of a usb cable)|

> [!WARNING]
> But I have a problem: with this wiring the door is physically kept unlocked (regardless of the ESPBell-LITE being powered or not). I have to disconnect the LOCK (red) --> OP to lock it.

Otherwise, the ESPBell-LITE is detected by home assistant, the bell is correctly detected.

**Comment by PricelessToolkit** [comment, marked as answer]
Hi, LOCK (black) and LOCK (red) are the same contacts "one side of the relay"

So, connect 6 (system ground) to "LOCK (black) and LOCK (red)" and the next two wires "gray and brown" to  "OP Door Opener"
